beacon-chain/core/helpers/randao.go

```diff
@@ -4,35 +4,42 @@ import (
 	"fmt"
 
 	pb "github.com/prysmaticlabs/prysm/proto/beacon/p2p/v1"
+	"github.com/prysmaticlabs/prysm/shared/bytesutil"
 	"github.com/prysmaticlabs/prysm/shared/hashutil"
 	"github.com/prysmaticlabs/prysm/shared/params"
 )
 
 // GenerateSeed generates the randao seed of a given epoch.
 //
 // Spec pseudocode definition:
-//   def generate_seed(state: BeaconState,
-//                  epoch: Epoch) -> Bytes32:
-//    """
-//    Generate a seed for the given ``epoch``.
-//    """
-//    return hash(
-//        get_randao_mix(state, epoch - MIN_SEED_LOOKAHED) +
-//        get_active_index_root(state, epoch)
-//    )
+// def generate_seed(state: BeaconState,
```

Comment:
Please format these to look the same as other `Spec pseudocode definition`

Reply:
thanks used godoc to check. lgtm after fix